openwisp_network_topology/management/commands/upgrade_from_django_netjsongraph.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+import uuid
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group, Permission
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+from swapper import get_model_name, load_model, split
+
+User = get_user_model()
+Topology = load_model('topology', 'Topology')
+Organization = load_model('openwisp_users', 'Organization')
+
+
+class BaseUpdateFromDjangoNetjsonGraph(BaseCommand):
+    help = 'Upgrade from django-netjsongraph'
+    app_label_users = split(get_model_name('openwisp_users', 'Organization'))[0]
+    app_label = split(get_model_name('topology', 'Topology'))[0]
+
+    def add_arguments(self, parser):
+        dirname = os.path.join(
```

Comment:
Can we just use `settings.BASE_DIR`?

Reply:
Thanks, it looks much better now! :smile: